metabase/metabase
# Use endpoints to fetch field values for card-based parameters

Epic https://github.com/metabase/metabase/issues/26898

How to test:
- New -> Native
- `SELECT * FROM PRODUCTS WHERE {{tag}}`
- Field Filter -> Products -> Category
- Dropdown -> Edit -> Custom list -> Add `Gadget` and `Widget`
- **Save the question**
- Click on the filter. It should now use a different BE endpoint and display only `Gadget` and `Widget`

Please note that this PR target the integration branch and not master.

<img width="999" alt="Screenshot 2023-01-20 at 16 48 16" src="https://user-images.githubusercontent.com/8542534/213727105-c7ceba8d-ddfe-468f-b1f5-bdcf89b6e881.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/27727)
